api/serializers/job_serializers.py

```diff
@@ -4,10 +4,14 @@
 
 
 class JobSerializer(serializers.ModelSerializer):
+    company_id = serializers.CharField(read_only=True, source='company.id')
```

Comment:
Considering it's more than a single field we're wanting to access from another table, we might as well just serialize the whole company using the CompanySerializer.